src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&family=Space+Mono&display=swap);
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logoLink {
+  display: block;
+  height: 40px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav__link {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin: 10px;
```

Comment:
Better to define `margin-right` or `margin-left`.
Also, the last element in list don't have margins at all